lib/model/image_sscd.py

```diff
@@ -0,0 +1,70 @@
+from typing import Dict
+import io
+import urllib.request
+
+from lib.model.model import Model
+
+from pdqhashing.hasher.pdq_hasher import PDQHasher
+from lib import schemas
+from torchvision import transforms
+from PIL import Image
+import torch
+from lib.logger import logger
+import requests
+import numpy as np
+
+class Model(Model):
+    def compute_sscd(self, image_url: str) -> str:
+        """Compute perceptual hash using ImageHash library
+        :param im: Numpy.ndarray
+        :returns: Imagehash.ImageHash
+        """
+        # pdq_hasher = PDQHasher()
+        # hash_and_qual = pdq_hasher.fromBufferedImage(iobytes)
+        # return hash_and_qual.getHash().dumpBitsFlat()
+        normalize = transforms.Normalize(
+            mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225],
+        )
+        small_288 = transforms.Compose([
```

Comment:
can you add some comments here about what is going on? and/or link to paper or docs about the implementation?

Reply:
@ahmednasserswe this is still to do

Reply:
@ahmednasserswe Can you please add the comments requested here?